packages/react/src/KeybindingHint/components/Sequence.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {Fragment} from 'react'
+import type {KeybindingHintProps} from '../props'
+import VisuallyHidden from '../../_VisuallyHidden'
+import {accessibleChordString, Chord} from './Chord'
+
+const splitSequence = (sequence: string) => sequence.split(' ')
+
+export const Sequence = ({keys, format = 'condensed', variant = 'normal'}: KeybindingHintProps) =>
+  splitSequence(keys).map((c, i) => (
+    <Fragment key={i}>
+      {
+        //  Since we audibly separate individual keys in chord with space, we need some other separator for chords in a sequence
+        i > 0 && (
+          <>
+            <VisuallyHidden>, then</VisuallyHidden>{' '}
```

Comment:
As far as I know, the normal spacing gets stripped by screen readers. I am not sure how that might look like though. 

On UnderlineNav, we have a similar practise and we use non-breaking space to make sure screen reasers announce the spacing properly.

https://github.com/primer/react/blob/main/packages/react/src/UnderlineNav/UnderlineNav.tsx#L331

I might be missing some context though - let me know 🙌🏻

Reply:
In the `UnderlineNav` case, the space falls at the end of an element before the closing tag. In this case, there's a space inside the element between text, and there's a space between two elements. I am pretty sure (like 85% sure 😅) sure these will work as expected given their placement, but I'll add some stories using `KeyboardHint` as an `aria-labelledby` target to see what actually happens.

Reply:
I actually noticed an _extra_ space in the accessible label, rather than a missing one 😄 - the AT is inserting a space between elements. So we'd get the label "x , then y". I solved this by removing the comma, which in retrospect feels like an unnecessary pause anyway - it reads pretty naturally as "x then y".